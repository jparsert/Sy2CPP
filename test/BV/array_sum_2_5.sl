; default grammar added
(set-logic BV)
(synth-fun findSum((y1 (_ BitVec 32))(y2 (_ BitVec 32)))
 (_ BitVec 32)
(( NTnonbool (_ BitVec 32))(NTbool Bool))
((NTnonbool (_ BitVec 32)(y1 y2 (_ bv0 32) (_ bv1 32) (_ bv2 32) (_ bv3 32)  (bvadd NTnonbool NTnonbool)(bvsub NTnonbool NTnonbool)(bvshl NTnonbool NTnonbool)(bvlshr NTnonbool NTnonbool)(bvurem NTnonbool NTnonbool)(ite NTbool NTnonbool NTnonbool)))
(NTbool Bool((and NTbool NTbool)(or NTbool NTbool)(not NTbool)(= NTnonbool NTnonbool)(bvuge NTnonbool NTnonbool)(bvugt NTnonbool NTnonbool)))

)
)

(declare-var x1 (_ BitVec 32))
(declare-var x2 (_ BitVec 32))
( constraint  (=> (bvugt (bvadd x1 x2) (_ bv5 32)) (= (findSum x1 x2 ) (bvadd x1 x2))) ) 
( constraint  (=> (bvule (bvadd x1 x2) (_ bv5 32)) (= (findSum x1 x2 ) (_ bv0 32))) ) 
(check-synth)
