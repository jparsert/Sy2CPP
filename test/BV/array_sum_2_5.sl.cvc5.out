cvc5 interrupted by SIGTERM.
cvc5 interrupted by SIGTERM.
Command terminated by signal 15
5.44user 0.00system 0:05.45elapsed 99%CPU (0avgtext+0avgdata 36636maxresident)k
0inputs+0outputs (0major+5074minor)pagefaults 0swaps
cvc5 interrupted by SIGTERM.
